File: src/main/kotlin/helpers/Logger.kt
package helpers

fun logRequest(msg: String) {
    println("[REQ] $msg")
}

fun logError(msg: String) {
    System.err.println("[ERR] $msg")
}

fun logInfo(msg: String) {
    println("[INFO] $msg")
}

fun createLogEntry(level: String, msg: String): Map<String, Any> {
    return mapOf("level" to level, "msg" to msg, "ts" to System.currentTimeMillis())
}

File: src/Helpers/logger.php
<?php

function log_request($msg) {
    echo '[REQ] ' . $msg . "\n";
}

function log_error($msg) {
    echo '[ERR] ' . $msg . "\n";
}

function log_info($msg) {
    echo '[INFO] ' . $msg . "\n";
}

function create_log_entry($level, $msg) {
    return ['level' => $level, 'msg' => $msg, 'ts' => time()];
}
